React/Views/RCTPicker.m

```diff
@@ -23,16 +23,17 @@ @implementation RCTPicker
 - (instancetype)initWithFrame:(CGRect)frame
 {
   if ((self = [super initWithFrame:frame])) {
-    _color = [RCTUIColor blackColor]; // TODO(OSS Candidate ISS#2710739)
 #if !TARGET_OS_OSX // TODO(macOS ISS#2323203)
     _font = [UIFont systemFontOfSize:21]; // TODO: selected title default should be 23.5
+    _color = [RCTUIColor blackColor]; // TODO(OSS Candidate ISS#2710739)
 #else // [TODO(macOS ISS#2323203)
-    _font = [UIFont systemFontOfSize:11];
+    _font = [UIFont systemFontOfSize:13];
```

Comment:
To ensure we reliability have the default font size as the OS evolves, can use `[NSFont systemFontOfSize:[NSFont systemFontSize]]`